resources/showIcons.php

```diff
@@ -58,7 +58,20 @@ function showFiles() {
     
     sort($list);
     foreach ($list as $entry) {
-        echo '<tr><td>'.$entry.' </td><td bgcolor="#C0C0C0"><a href="'.$entry.'"><img src="'.$entry.'"></a></td>'."\n";
+        echo '<tr>'."\n";
+        echo     '<td><a href="'.$entry.'">'.$entry.' </a></td>'."\n";
+       if (endswith($entry, ".gif") 
+            || endswith($entry, ".jpg") 
+            || endswith($entry, ".png")
+            ) {
+                // display as image
+                //   would be good to add a limiting size here
+                echo     '<td bgcolor="#C0C0C0"><a href="'.$entry.'"><img src="'.$entry.'"></a></tr>'."\n";
```

Comment:
Thanks! Didn't know about that. 

Does it preserve the aspect ratio? If not, should there also be a max-height attribute?